internal/manager/route.go

```diff
@@ -115,8 +116,6 @@ var (
 		"gpt-4-32k-0613",
 		"gpt-4-32k-0314",
 		"gpt-4-turbo",
-		"gpt-4o-2024-05-13",
```

Comment:
we should not remove models due to backward compatibility

Reply:
the element is redundant. It's already been declared at the top of the list.